### PR TITLE
feat: new mux quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New Mux quickstart. Includes Dashboard, data source, and quickstart files. Creating as a draft until the docs are published.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by aswanson-nr — DISMISSED** [review]

**Comment by jcountsNR** [comment]
@aswanson-nr has this been merged yet? I'm thinking we should hold off for just a bit, I got some feedback from Mux and they want to make a few changes to the dashboard and text here.

**Comment by aswanson-nr** [comment]
Not stale! going to remove that label.
@jcountsNR this appears to be ready to go, but there's a `test.json` dashboard. If that's intended to be part of this quickstart, could adjust the name to be more descriptive?

**Comment by brammerl** [comment]
Removing the `stale` logo. Waiting on JRCounts to get clearance before publishing.

**Comment by jcountsNR** [comment]
@aswanson-nr / @seemantk - Updating this one. It should be ready to be reviewed in the next cycle!

File: quickstarts/mux-video/config.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  Check out the [documentation](https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/) to learn more about setting up New Relic monitoring for of data from Mux.
+summary: |
+  Mux is video infrastructure that makes it easy for development teams to get actionable data at industry-leading scale, and build video into their products quickly and reliably.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic (Joseph Counts)
+title: Mux Video Analytics (via Amazon Firehose)
+installPlans:
```
Comment (brammerl) [inline quickstarts/mux-video/config.yml:21]: we are not longer allowing installPlans. Please delete the install plan you made and this reference to installPlans.

File: data-sources/mux/config.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+id: mux-video
+displayName: Mux
+description: |
+  Mux allows users to launch video features in days, customize the player experience, and monitor video streaming performance, all while scaling seamlessly to the world’s largest audiences.
+icon: logo.svg
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/
+keywords:
```
Comment (brammerl) [inline data-sources/mux/config.yml:10]: Same with these keywords. A lot of them aren't on the list of automatically accepted keywords. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#keywords

@seemantk

File: quickstarts/mux-video/config.yml (hunk 1 above)
Comment (jcountsNR) [reply quickstarts/mux-video/config.yml]: done.

File: dashboards/test/test.json [hunk 3]
```diff
@@ -0,0 +1,413 @@
+{
+    "name": "Mux Video Streaming",
```
[the remark this answers is not in the document]
Comment (jcountsNR) [reply dashboards/test/test.json]: this was a mistake, it should not have merged. I deleted it.

File: quickstarts/mux-video/config.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  - third-party-mux
+dataSourceIds:
+  - mux-video
+dashboards:
+  - mux-video-analytics
+documentation:
+  - name: Mux installation docs
+    description: Send Mux logs to New Relic using Amazon Firehose
+    url: https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/
+keywords:
```
[the remark this answers is not in the document]
Comment (seemantk) [reply quickstarts/mux-video/config.yml:29]: this is an interesting one.  I don't have any strong objections to the non-traditional ones, but I would love to see if the new ones ("video" and "streaming") can apply to other existing or upcoming quickstarts

File: data-sources/mux/config.yml (hunk 2 above)
Comment (seemantk) [reply data-sources/mux/config.yml:10]: "quality of experience" is the only one that I'm not a huge fan of here, but the rest seem fine to me

**Review by brammerl — APPROVED** [review]

**Pull request merged**